warehouse/query-core/src/test/java/datawave/query/jexl/nodes/QueryPropertyMarkerTest.java

```diff
@@ -57,13 +58,13 @@ public void testInstance_of() throws ParseException {
     public void testInstance_isAnyType() throws ParseException {
         assertFalse(QueryPropertyMarker.Instance.of().isAnyType());
         
-        QueryPropertyMarker.Instance instance = QueryPropertyMarker.Instance.of(ASTDelayedPredicate.class, parseJexlQuery("FOO == 'a'"));
+        QueryPropertyMarker.Instance instance = QueryPropertyMarker.Instance.of(ASTDelayedPredicate.class, Lists.newArrayList(parseJexlQuery("FOO == 'a'")));
```

Comment:
This implies we could have a chained constructor for Instance.of()
```
Instance.of(JexlNode node){
    this(Lists.newArrayList(node))
}
...
Instance.of(Collection<JexlNode> nodes)
```

Reply:
Added!